api/db-cdr/generate-cdr/make-bq-public-data.sh

```diff
@@ -53,47 +39,20 @@ then
 fi
 
 # Check that source dataset exists and exit if not
-datasets=$(bq --project=$WORKBENCH_PROJECT ls)
-if [ -z "$datasets" ]
-then
-  echo "$WORKBENCH_PROJECT.$WORKBENCH_DATASET does not exist. Please specify a valid project and dataset."
-  exit 1
-fi
-re=\\b$WORKBENCH_DATASET\\b
-if [[ $datasets =~ $re ]]; then
-  echo "$WORKBENCH_PROJECT.$WORKBENCH_DATASET exists. Good. Carrying on."
-else
-  echo "$WORKBENCH_PROJECT.$WORKBENCH_DATASET does not exist. Please specify a valid project and dataset."
-  exit 1
-fi
-
-# Check that public project exists
 datasets=$(bq --project=$PUBLIC_PROJECT ls)
 if [ -z "$datasets" ]
 then
-  echo "$PUBLIC_PROJECT does not exist. Please specify a valid PUBLIC project and dataset."
+  echo "$PUBLIC_PROJECT.$PUBLIC_DATASET does not exist. Please specify a valid project and dataset."
   exit 1
 fi
-
-# Make dataset for public cloudsql tables
-datasets=$(bq --project=$PUBLIC_PROJECT ls)
 re=\\b$PUBLIC_DATASET\\b
 if [[ $datasets =~ $re ]]; then
-  echo "$PUBLIC_DATASET exists"
+  echo "$PUBLIC_PROJECT.$PUBLIC_DATASET exists. Good. Carrying on."
 else
-  echo "Creating $PUBLIC_DATASET"
-  bq --project=$PUBLIC_PROJECT mk $PUBLIC_DATASET
+  echo "$PUBLIC_PROJECT.$PUBLIC_DATASET does not exist. Please specify a valid project and dataset."
+  exit 1
 fi
 
-
-copy_tables=(achilles_analysis achilles_results achilles_results_dist concept concept_relationship criteria criteria_attribute domain_info survey_module domain vocabulary concept_synonym domain_vocabulary_info)
```

Comment:
I noticed that this list isn't actually exhaustive, there were maybe 3 tables that weren't being copied. Will this cause any issues if those extra tables now wind up in the public BQ datasets?

Reply:
The data generation did not complain. They would probably sit there unused after the count generation. Except that, i have a concern now with person_gender_identity being there without any need. I am going to update the script to remove the unused ones at the end of data generation. Thanks.